### ISSUE TITLE
In eval step, can delay 2 hour!!! how can i tune for hurry this step?

### ISSUE THREAD

**Issue by Illanes** [TimDettmers/bitsandbytes#174]
That is, colab has set eval each 500 steps, when reach to 501, carry out the first, but has taken 2 or more hour and colab is disconected, so how can i tune it for hurry eval? 

from transformers import Seq2SeqTrainingArguments

training_args = Seq2SeqTrainingArguments(
    output_dir="temp",  # change to a repo name of your choice
    per_device_train_batch_size=8,
    gradient_accumulation_steps=1,  # increase by 2x for every 2x decrease in batch size
    learning_rate=1e-3,
    warmup_steps=50,
    max_steps=1500,
    evaluation_strategy="steps",
    fp16=True,
    per_device_eval_batch_size=8,
    generation_max_length=128,
    eval_steps=500,
    logging_steps=25,
    remove_unused_columns=False,  # required as the PeftModel forward doesn't have the signature of the wrapped model's forward
    label_names=["labels"],  # same reason as above
)

After 6 Hours of training, its stop, with no errors, just stop, with a warning

![image](https://user-images.githubusercontent.com/7408622/222527421-41373f8c-4a48-45e4-9488-75a8c710111e.png)
